src/tests/multihost/ad/test_access_control.py

```diff
@@ -0,0 +1,400 @@
+""" AD access control test ported from bash
+:requirement: ad_access_control
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:status: approved
+"""
+import pytest
+import re
+import pexpect
+import time
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.exceptions import SSHLoginException
+
+
+def ssh_login(multihost, username):
+    client_hostname = multihost.client[0].sys_hostname
+    client = pexpect_ssh(client_hostname, username, 'Secret123', debug=False)
+    cmd = 'echo > /var/log/secure'
+    multihost.client[0].run_command(cmd, raiseonerr=False)
+    try:
+        client.login()
+    except SSHLoginException:
+        return 'false', 'login failed'
+    except pexpect.EOF:
+        time.sleep(1)
+        log_str = multihost.client[0].get_file_contents('/var/log/secure').decode('utf-8')
+        patt = re.compile(r'Access.*denied for user')
+        return 'denied', patt.findall(log_str)
+    else:
+        (stdout, _) = client.command(f'id {username}')
+        client.logout()
+        return 'Success', stdout
+
+
+@pytest.mark.usefixtures('joinad')
+@pytest.mark.ad_access_control
+@pytest.mark.tier1_4
+class TestAccessControl(object):
+    """ Test cases for BZ: 1268902
+    :setup:
+        1. Join to AD using realm command.
+        2. Add the user using adcli user add.
+    """
+    @staticmethod
+    def test_001_simple_allow_user_to_user1(multihost, create_aduser_group, backupsssdconf):
+        """
```

Comment:
Please make sure that the test docssting has the decription starting on first line and a blank line before other fields.

Reply:
Blank line added after first line in the file